interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/xscobot_moveit.launch.py

```diff
@@ -299,9 +300,13 @@ def launch_setup(context, *args, **kwargs):
     )
 
     return [
-        move_group_node,
-        moveit_rviz_node,
         xscobot_ros_control_launch_include,
+        TimerAction(
+                period=5.0,
+                actions=[move_group_node,
+                         moveit_rviz_node,
+                        ],
+                    ),
         # xsarm_gz_classic_launch_include,
```

Comment:
> `# xsarm_gz_classic_launch_include`

Commented line?

Reply:
This is for gazebo node in case we add that in the future. Do we still want it removed?

Reply:
Yeah, remove it for now.